### PR TITLE
ETK: Move phpunit to TeamCity

### PR DESCRIPTION
**Pull request by noahtallen**
#### Changes proposed in this Pull Request
We need to see by how much this increases the TeamCity build. Theoretically, it should normally take about a minute to create the WordPress instance. (We can also investigate caching the wp-env home directory.)

#### Testing instructions
- No more ETK GitHub actions.
- PHPunit step in TeamCity ETK build should pass eventually.

### CODE REVIEW

File: Dockerfile.base [hunk 1]
```diff
@@ -137,7 +137,9 @@ RUN apt update &&\
 	wget https://packages.sury.org/php/apt.gpg -O /etc/apt/trusted.gpg.d/php-sury.gpg &&\
 	echo "deb https://packages.sury.org/php/ buster main" > /etc/apt/sources.list.d/php-sury.list &&\
 	apt update &&\
-	apt-get install -y php7.4-cli php7.4-xml php7.4-mbstring docker-compose &&\
-	composer install
+	apt-get install -y php7.4-cli php7.4-xml php7.4-mbstring &&\
+	composer install &&\
+	sudo curl -L --fail https://github.com/docker/compose/releases/download/1.28.6/run.sh -o /usr/local/bin/docker-compose &&\
```
Comment (noahtallen) [inline Dockerfile.base]: copied from https://docs.docker.com/compose/install/#install-as-a-container

**Comment by noahtallen** [comment]
@scinos the version of docker-compose installed by `apt` seems to be [1.21](https://docs.docker.com/compose/release-notes/#1210), which is about 3 years old. The curl approach is suggested by the docks, so I'm going to see if that at least fixes it

**Comment by noahtallen** [comment]
The current approach fails with:

```
/usr/local/bin/docker-compose: line 91: exec: docker: not found
Process exited with code 127
```

Not sure if anyone has any ideas about that. This is starting to get complicated and the docs are not very good. This just worked out of the box on Circle, Travis, and GitHub actions. I think their Docker images include `docker-compose` by default. I may pause this work for a while.

**Comment by scinos** [comment]
> @scinos the version of docker-compose installed by apt seems to be 1.21, which is about 3 years old

Right. Looks like `wp-env` is using [`version: 3.7` of `docker-composer.yml`](https://github.com/WordPress/gutenberg/blob/trunk/packages/env/lib/build-docker-compose-config.js#L174), and that version [requires at least `docker-compose 1.22`](https://docs.docker.com/compose/release-notes/#1220)

**Comment by noahtallen** [comment]
Closing for now, but hope to return to it.

**Pull request closed without merging**